pytorch_lightning/loggers/tensorboard.py

```diff
@@ -29,9 +29,11 @@ class TensorBoardLogger(LightningLoggerBase):
 
     Args:
         save_dir (str): Save directory
-        name (str): Experiment name. Defaults to "default".
-        version (int): Experiment version. If version is not specified the logger inspects the save
+        name (str): Experiment name. Defaults to "default".  If it is the empty string then no per-experiment
+        subdirectory is used.
+        version (int|str): Experiment version. If version is not specified the logger inspects the save
         directory for existing versions, then automatically assigns the next available version.
```

Comment:
this should have extra tab spacing as it belongs as a block to an argument

Reply:
Done